src/project.coffee

```diff
@@ -65,15 +65,15 @@ class Project extends Model
           fs.closeSync(fs.openSync(bufferState.filePath, 'r'))
         catch error
           return unless error.code is 'ENOENT'
-      deserializerManager.deserialize(bufferState)
+      TextBuffer.deserialize(bufferState)
```

Comment:
Here we can get rid of the `DeserializerManager` because we have removed the `Serializable` mixin in `TextBuffer` on https://github.com/atom/text-buffer/pull/136